meteor/client/ui/RundownList.tsx

```diff
@@ -185,6 +186,8 @@ export function RundownList(): JSX.Element {
 				)}
 			</section>
 
+			{getAllowStudio() && <CreateTestingRundownPanel />}
```

Comment:
About naming of this feature:
An idea is to name this "Adlib Rundowns", since that would align with the idea of "unplanned things, created/played/instanciated by the user"